cli/README.md

```diff
@@ -0,0 +1,46 @@
+A command-line interface for interfacing with Immich
```

Comment:
Can you please add some notes for how to use this in development? ts-node is not a dev-dependency, correct? You would have to install it separately

Reply:
I must admit I'm not knowledgeable enough with ts to know the best way to run a script. Perhaps you can advise?

Reply:
`npx ts-node cli/src` would probably work. I think,  though, eventually all these notes should get moved to the documentation side, right?

Reply:
Yes, they will as soon as we feel this one is read